phpBB/phpbb/user_loader.php

```diff
@@ -189,12 +189,7 @@ public function get_avatar($user_id, $query = false, $lazy = false)
 			return '';
 		}
 
-		if (!function_exists('get_user_avatar'))
-		{
-			include($this->phpbb_root_path . 'includes/functions_display.' . $this->php_ext);
-		}
-
-		return get_user_avatar($user['user_avatar'], $user['user_avatar_type'], $user['user_avatar_width'], $user['user_avatar_height'], 'USER_AVATAR', false, $lazy);
+		return phpbb_get_avatar(\phpbb\avatar\manager::clean_row($user, 'user'), 'USER_AVATAR', false, $lazy);
```

Comment:
@marc1706 learn to love the wrapper :smile: 
![giphy](https://cloud.githubusercontent.com/assets/303711/11567477/0e52777e-999c-11e5-81f9-405786b1a066.gif)